snooty.toml

```
@@ -79,6 +79,7 @@ max-graphql-resolvers = "ten"
 realm = "realm"
 realms = "realms"
 realm-web = "Realm Web SDK"
+request-timeout-time = "120 seconds"
```

```suggestion
request-timeout-time = "150 seconds"
```
Looks like you forgot to actually change the timeout to 150

---

😓